File: library/src/main/java/com/github/dmitrikudrenko/countryphonecodechooser/ui/adapter/sections/Section.java
package com.github.dmitrikudrenko.countryphonecodechooser.ui.adapter.sections;

import android.support.annotation.RestrictTo;
import android.support.annotation.RestrictTo.Scope;

@RestrictTo(Scope.LIBRARY)
public class Section {
    public static final Section EMPTY_SECTION = new Section(' ');

    private final char id;

    Section(final char id) {
        this.id = id;
    }

    public char getId() {
        return id;
    }
}

File: library/src/main/kotlin/com/github/dmitrikudrenko/countryphonecodechooser/ui/adapter/sections/Section.kt
package com.github.dmitrikudrenko.countryphonecodechooser.ui.adapter.sections

import android.support.annotation.RestrictTo
import android.support.annotation.RestrictTo.Scope

@RestrictTo(Scope.LIBRARY)
data class Section internal constructor(val id: Char) {
    companion object {
        val EMPTY_SECTION = Section(' ')
    }
}
